testing/functional/tests/cypress/fixtures/instituteManageLocationData.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please use the word Institution instead of Institute as it is used all over the system.

Reply:
Changes are done.

Reply:
I still can see files and code using the word Institute. Was it changed across the entire PR?

Reply:
Changes are done & updated across institution section.

Reply:
Can you please do it in the entire PR?
![image](https://user-images.githubusercontent.com/61259237/163225590-90698aad-b51c-4a51-a05f-0cf6f6d4cb4f.png)

Reply:
Check in latest synced.